pkg/api/2019-10-27-preview/converterfrominternal.go

```diff
@@ -34,6 +34,7 @@ func FromInternal(cs *api.OpenShiftManagedCluster) *OpenShiftManagedCluster {
 		ClusterVersion:    &cs.Properties.ClusterVersion,
 		PublicHostname:    &cs.Properties.PublicHostname,
 		FQDN:              &cs.Properties.FQDN,
+		PrivateAPIServer:  &cs.Properties.PrivateAPIServer,
```

Comment:
Was the final decision to expose this at the top level? IMO this shouldn't be a top level property.

Reply:
I am happy to move it (to MasterAgentPool?)

Reply:
I have now moved it to the MasterPoolProfile